lang/ruby/lib/avro/schema.rb

```diff
@@ -137,6 +137,18 @@ def initialize(type)
     # Deprecated in favor of {#type_sym}.
     def type; @type_sym.to_s; end
 
+    # Returns the MD5 fingerprint of the schema as an Integer.
+    def md5_fingerprint
```

Comment:
might it make sense to memoize these? there are a couple other fields which are memoized, and it would make sense with the use case of generating several records from a schema and including the fingerprint in each